common/config/rush/pnpm-lock.yaml

```diff
@@ -1057,33 +1037,6 @@ packages:
     resolution: {integrity: sha512-HbE7iORnYcjLzKNf5mIQRJQDTsVxhoXHRWEZ6KWdGh4e7+F9xTloiBicavbSoVmlAYivenIVpryHanVwsQaHUw==}
     dev: false
 
-  /@azure/identity/1.2.5_debug@4.3.4:
```

Comment:
It seems that our lock introduces unnecessary change which caused the core ci error https://dev.azure.com/azure-sdk/public/_build/results?buildId=1925438&view=results.

Could you follow up steps to revert them?

```
> git fetch origin main  // get the latest main branch
> git checkout origin/main common/config/rush/pnpm-lock.yaml // checkout the lock file
> rush update // only apply our change
```